dGame/dComponents/PetComponent.cpp

```diff
@@ -884,7 +888,7 @@ void PetComponent::Activate(Item* item, bool registerPet, bool fromTaming) {
 
 	EntityManager::Instance()->SerializeEntity(m_Parent);
 
-	owner->GetCharacter()->SetPlayerFlag(69, true);
```

Comment:
I will note, the specific description of this flag as we have it is "First manual pet hibernate" and not "Activated Pet".   This may be confusing to future maintainers having a mis-match

Reply:
will fix

Reply:
Also, I want to eventually go through and defined all of these that are just magic numbers scattered throughout the codebase. but that was a larger undertaking that I decided not to do in this PR. Could you point me in the direction of where we have those flags?